colorfield/static/colorfield/colorfield.js

```diff
@@ -1,12 +1,16 @@
-/** global: django */
-
-window.addEventListener('load', function (event) {
-    if (typeof django !== 'undefined' && typeof django.jQuery !== 'undefined') {
-        (function ($) {
-            // add colopicker to inlines added dynamically
-            $(document).on('formset:added', function onFormsetAdded(event, row) {
-                jscolor.install();
-            });
-        })(django.jQuery);
+window.addEventListener('load', function () {
+  const inputs = document.getElementsByClassName('colorfield_field coloris');
+  for (const input of inputs) {
+    try {
+      const id = input.getAttribute('id');
+      // convert python dict into json object
+      const options = input.getAttribute('data-coloris')
+        .replaceAll("'", '"')
+        .replaceAll('True', 'true')
+        .replaceAll("False", 'false');
+      Coloris.setInstance(`.colorfield_field.coloris.${id}`, JSON.parse(options));
+    } catch (_ignore) {
+      console.debug('cannot read data-coloris attribute in input', id);
```

Comment:
This should never happen, we don't need a try/catch block here, are you using AI? :)

Reply:
It did happen during my tests when I was deciding and testing the best approach ;)

Reply:
In case it would be better to understand the cause and prevent it instead of catching an unexpected error.

Reply:
Let's say it was a leftover...